x/group/client/util.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func parseMembers(membersFile string) ([]group.Member, error) {
+	members := []group.Member{}
+
+	if membersFile == "" {
+		return members, nil
+	}
+
+	contents, err := ioutil.ReadFile(membersFile)
+	if err != nil {
+		return nil, err
+	}
+
+	err = json.Unmarshal(contents, &members)
```

Comment:
Given that `group.Member` only has string subfields, encoding/json works here, but technically it should be clientCtx.JSONMarshaler.UnmarshalJSON, right?

Reply:
good point

Reply:
the thing is that `members` a just a regular slice, not `proto.Message` so I guess we might need some `Members` proto type in this case

Reply:
ah yeah. That's annoying about protobuf, if you want an array of something you need to create a struct with an array subfield.